packages/@webex/plugin-meetings/src/multistream/mediaRequestManager.ts

```diff
@@ -175,6 +182,52 @@ export class MediaRequestManager {
     );
   }
 
+  /**
+   * Returns the maxPayloadBitsPerSecond per Stream
+   *
+   * If MediaRequestManager kind is "audio", a constant bitrate will be returned.
+   * If MediaRequestManager kind is "video", the bitrate will be calculated based
+   * on maxFs (default h264 maxFs as fallback if maxFs is not defined)
+   *
+   * @param {MediaRequest} mediaRequest  - mediaRequest to take data from
+   * @returns {number} maxPayloadBitsPerSecond
+   */
+  private getMaxPayloadBitsPerSecond(mediaRequest: MediaRequest): number {
+    if (this.kind === 'audio') {
+      // return mono_music bitrate default if the kind of mediarequest manager is audio:
+      return RecommendedOpusBitrates.FB_MONO_MUSIC;
```

Comment:
@marcin-bazyl, this was suggested by me as we're using 64 KB now.
Opus codec format will be always opus/48000/2 in sdp. the fmtp param stereo indicates whether it prefer to receive stereo but acutally it can always decode both mono/stereo. can find details here - https://www.rfc-editor.org/rfc/rfc7587.html